packages/runtime/src/webcontainer/terminal-config.spec.ts

```diff
@@ -144,13 +144,13 @@ describe('TerminalConfig', () => {
     expect(config.panels[0].title).toBe('TERM 1');
     expect(config.panels[0].processOptions).toEqual({
       allowRedirects: true,
-      allowCommands: ['echo'],
+      allowCommands: ['ls', 'echo'],
```

Comment:
This (and change below) doesn't look right. Above we are setting `allowCommands: ['echo']`. 

```ts
134 |    const config = new TerminalConfig({
135 |      allowRedirects: true,
136 |      allowCommands: ['echo'],          // <--
```

Here on the assertion we should see just `echo`, no `ls`.

```suggestion
      allowCommands: ['echo'],
```